repo/packages/K/kubernetes/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "kubernetes",
+  "version": "k8s-0.14.2-k8sm-0.5-dcos-20150514T1802520000",
+  "scm": "https://github.com/mesosphere/kubernetes-mesos",
+  "maintainer": "support@mesosphere.io",
+  "description": "Manage a cluster of Linux containers as a single system to accelerate Dev and simplify Ops.",
+  "framework": true,
+  "images": {
+    "icon-small": "https://downloads.mesosphere.io/kubernetes/artifacts/images/k8s-48x48.png",
+    "icon-medium": "https://downloads.mesosphere.io/kubernetes/artifacts/images/k8s-96x96.png",
+    "icon-large": "https://downloads.mesosphere.io/kubernetes/artifacts/images/k8s-256x256.png"
+  },
+  "tags": ["mesosphere", "framework", "pod", "label", "paas"],
+  "preInstallNotes": "In order for Kubernetes to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 1 node with 1 CPU share and 128 MB of RAM available for use by the Kubernetes service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
+  "postInstallNotes": "Kubernetes DCOS Service has been successfully installed!\n\n\tDocumentation: https://github.com/mesosphere/kubernetes-mesos\n\tIssues: https://github.com/mesosphere/kubernetes-mesos/issues",
+  "licenses": [
+    {
+      "name": "Apache License Version 2.0",
+      "url": "https://github.com/mesosphere/kubernetes-mesos/blob/master/LICENSE"
+    }
+  ]
```

Comment:
Can we also add postUninstallNotes here which simply link to our uninstall doc, @jdef?